plugins/scalapb/scalapb/v0.11.12/buf.plugin.yaml

```
@@ -0,0 +1,9 @@
+version: v1
+name: buf.build/scalapb/scalapb
```

```suggestion
name: buf.build/community/scalapb-scala
```

For context, The Buf team doesn't own this owner namespace so we won't push a plugin there. For such scenarios, we have a `buf.build/community` organization. The naming convention for the plugin is `{owner}-{name}`.

Where `repository` is usually the upstream github owner, and the name is usually the name of the plugin minus `protoc-gen-` prefix.

---

Gorcha, that makes sense. I also moved the directory based on that naming scheme as well.